cloud/deployment/inspect/inspect.go

```diff
@@ -166,14 +166,17 @@ func getDeploymentInfo(coreDeployment astroplatformcore.Deployment) (map[string]
 	if err != nil {
 		return nil, err
 	}
-	clusterID := coreDeployment.ClusterId
+	var clusterID string
 	releaseName := coreDeployment.Namespace
 	if deployment.IsDeploymentStandard(*coreDeployment.Type) || deployment.IsDeploymentDedicated(*coreDeployment.Type) {
-		if deployment.IsDeploymentStandard(*coreDeployment.Type) {
-			clusterID = nil
-		}
 		releaseName = notApplicable
 	}
+	if !deployment.IsDeploymentStandard(*coreDeployment.Type) {
```

Comment:
can we add/update unit tests for both these cases as well

Reply:
We already have unit tests coving each deployment type

Reply:
I was wondering why this bug wasn't identified earlier via the unit tests. It might be beneficial for us to consider adding a test case for every bug we resolve, to help prevent any regressions in the future. What do you think?